tests/validation/test_validation.py

```diff
@@ -75,8 +76,9 @@ def test_mocked_validation_n(validator):
 def test_google_multi_n():
     candidate = Candidate(
         line_data_list=[  #
-            LineData({}, line="dummy line 1", line_num=1, path="dummy path 1", pattern=regex.compile('.*')),
-            LineData({}, line="dummy line 2", line_num=2, path="dummy path 2", pattern=regex.compile('.*'))
+            LineData({}, line="dummy line 1", line_num=1, path="dummy path 1", info="info",
```

Comment:
This is a strange formatting. The following line is almost the same but its format differs.

Reply:
over 120 symbols possibly